.github/workflows/publish-types.yaml

```diff
@@ -0,0 +1,28 @@
+name: Publish Types
+
+on:
+  release:
+    types: [created, published]
+  push:
+    branches:
+      - staging
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Publish types
+        run: cd types/generated && npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```

Comment:
I'll need to add this in the repo secret.

Reply:
Added.

Reply:
ok